libpod/networking.go

```diff
@@ -102,3 +108,63 @@ func (r *Runtime) teardownNetNS(ctr *Container) error {
 
 	return nil
 }
+
+//ReadPortRangeFile reads the range of available ports per proc
+func ReadPortRangeFile() (int, int, error) {
+	portFileName := "/proc/sys/net/ipv4/ip_local_port_range"
+	portFile, err := ioutil.ReadFile(portFileName)
+	if err != nil {
+		return 0, 0, errors.Wrapf(err, "unable to read %s", portFileName)
+	}
+	portFields := bytes.Fields(portFile)
+	if len(portFields) < 2 {
+		return 0, 0, fmt.Errorf("invalid data in %s", portFileName)
```

Comment:
Lets return an error libpod.InvalidData or something like that.

Reply:
ok, add and fixed